templates/docs/patterns/heading-icon.md

```diff
@@ -31,7 +31,15 @@ The icon for this component is also available at a smaller size of 32 × 32 pixe
 View example of the pattern heading icon small
 </a></div>
 
-## Import
+### Muted
```

Comment:
We updated heading levels last week and this seems like a conflict resolution.

Both "Muted" and "Import" should be 2nd level heading.
```suggestion
## Muted
```